Nodejs/Setup/NodejsTools/NodejsToolsFiles.proj

```diff
@@ -17,6 +17,7 @@
         <File Include="Microsoft.NodejsTools.ProjectWizard.dll" />
         <File Include="Microsoft.NodejsTools.Analysis.dll" />
         <File Include="Microsoft.NodejsTools.TestAdapter.dll" />
+        <File Include="Microsoft.NodejsTools.Telemetry.dll" />
```

Comment:
Is there anything else that needs to be added to the installer here? (Aren't we dependent on an external dll in the event that we don't fall back to Dummy?)